### ISSUE TITLE
Video stops in less than 2 minutes

### ISSUE THREAD

**Issue by hejazb** [NapoleonWils0n/cerberus#6]
Dear: Napoleon
I have a question and I do not understand the Kodi video player behavior.
The problem is that video stops playing after short period of time and almost always t shows the same short clip. The source is not encrypted, it is not pay preview and includes advertising and in some cases it is advertising site. This is regarding almost all Persian (Farsi) sites which some have their own web site.
In Phoenix then to Crusader88 -> International channels -> Iranian Channels, almost all behave the same, or if you install GLWIZ and then select TV channels again it is the same situation.
I have added advancedsettings.xml file and increased buffering time, used VPN and different proxy servers and manipulated Kodi video and system settings but the videos stop after a short while, most of the times clips are the same no matter when you play them.
Interesting, is that Sima Azadi “ www.iranntv.com” that you helped me to make it work in play list loader, works fine 24/7. I am using Kodi 16.1 32 bit, Windows 8.1 Pro 64 bit, Windows 10 64 and 32 bit and Android 5 64 and 32 bit devices, all behave the same. There are two web portals “iranproud.com” and “mellitv.com” they relay these Farsi stations but they charge almost $100 a year something that is free to watch on many satellites around the world and on the web.
Please, shed some light, may be you have a magic as always. Thank you for your precious work.
I wish you best.
Harry

**Comment by NapoleonWils0n** [comment]
Hi Harry

Missed your comment here, sorry for the delay

I have seen this before, 
when you open a link it always plays the same video and starts at the video at the same time and then stops after a few minutes

Here's what i think the problem is

Normally live tv streams are segmented ( chopped up ) into lots of video.ts files in real time, 
then you have a m3u8 playlist that has links to those video.ts files.

As the video is streaming new video.ts file are created and then the links to those video.ts files are added to the m3u8 playlist

So the m3u8 playlist is updated in realtime, links to old video.ts files are removed and links to new video.ts files are added as they are created by the streaming server

There are usually links to about a dozen or so video.ts files in the m3u8 playlist at any one time, but as the video streams its updated and old links are removed and new ones added

The video.ts file are usually about 30 seconds or so in length so a m3u8 playlist that isnt updated only lasts about 5-10 minutes because it has links to 12 video.ts file that are only 30 seconds in length

I think the m3u8 playlists are just links to some video.ts files and arent the real live tv streams. 

The m3u8 playlist always have the links to the same old video.ts files, thats why the always play the same video and only last a couple of minutes, because new video.ts files arent being add to the m3u8 playlist

basically the m3u8 playlist are just part of a live video stream and only contain a couple of video.ts files that last a few minutes, normally tv streams would have hundreds or thousands of video.ts file

So it not you mate, its the m3u8 playlists
Will need to find another source unfortunately
